server/constants/messages.go

```diff
@@ -3,22 +3,23 @@ package constants
 const (
 	// TODO: all these messages are to be verified from Mike at the end
 	// Generic
-	GenericErrorMessage            = "Something went wrong, please try again later"
-	SessionExpiredMessage          = "Session expired. Please connect your Azure DevOps account again"
-	ConnectAccount                 = "[Click here to connect your Azure DevOps account](%s%s)"
-	ConnectAccountFirst            = "Your Azure DevOps account is not connected \n%s"
-	UserConnected                  = "Your Azure DevOps account is successfully connected!"
-	UserAlreadyConnected           = "Your Azure DevOps account is already connected"
-	UserDisconnected               = "Your Azure DevOps account is now disconnected"
-	CreatedTask                    = "Work item [#%d: \"%s\"](%s) of type \"%s\" was successfully created by %s."
-	TaskTitle                      = "[%s #%d: %s](%s)"
-	PullRequestTitle               = "[#%d: %s](%s)"
-	BuildDetailsTitle              = "[#%s](%s): %s"
-	PipelineDetailsTitle           = "[%s](%s): %s"
-	AlreadyLinkedProject           = "This project is already linked."
-	NoProjectLinked                = "No project is linked, please link a project."
-	PipelinesRequestBeingProcessed = "Your approval/rejection request is being processed."
-	PipelinesRequestProcessed      = "Your approval/rejection request is processed."
+	GenericErrorMessage                            = "Something went wrong, please try again later"
+	SessionExpiredMessage                          = "Session expired. Please connect your Azure DevOps account again"
+	ConnectAccount                                 = "[Click here to connect your Azure DevOps account](%s%s)"
+	ConnectAccountFirst                            = "Your Azure DevOps account is not connected \n%s"
+	UserConnected                                  = "Your Azure DevOps account is successfully connected!"
+	MattermostUserAlreadyConnected                 = "Your Azure DevOps account is already connected"
+	UserDisconnected                               = "Your Azure DevOps account is now disconnected"
+	CreatedTask                                    = "Work item [#%d: \"%s\"](%s) of type \"%s\" was successfully created by %s."
+	TaskTitle                                      = "[%s #%d: %s](%s)"
+	PullRequestTitle                               = "[#%d: %s](%s)"
+	BuildDetailsTitle                              = "[#%s](%s): %s"
+	PipelineDetailsTitle                           = "[%s](%s): %s"
+	AlreadyLinkedProject                           = "This project is already linked."
+	NoProjectLinked                                = "No project is linked, please link a project."
+	PipelinesRequestBeingProcessed                 = "Your approval/rejection request is being processed."
+	PipelinesRequestProcessed                      = "Your approval/rejection request is processed."
+	ErrorMessageAzureDevopsAccountAlreadyConnected = "azure devops account for %s is already connected"
```

Comment:
I think this should be with the other error messages in the below group of const variables